hubblestack_nova/modules/win_secedit.py

```diff
@@ -91,6 +91,10 @@ def audit(data_list, tags, verbose=False):
                         secedit_ret = _get_operation(secedit_ret,
                                                      tag_data['value_type'],
                                                      tag_data['match_output'])
+                    if 'priv' in tag_data['value_type']:
+                        if 'no one' in tag_data['match_output']:
+                            if 'Not Defined' or None in secedit_ret:
```

Comment:
This is not quite right. It will treat this as two statements `'Not Defined'` and `None in secedit_ret`, and then `or` the results. Because non-empty strings are considered True, `'Not Defined'` is True, and so the if statement will always succeed.

Try this instead:

```
if 'Not Defined' in secedit_ret or None in secedit_ret:
```

Of course that still might not be quite right. `None` cannot be in a string. Perhaps you meant `None == secedit_ret`?